### ISSUE TITLE
Empty screen after opening app

### ISSUE THREAD

**Issue by Bodenseematze** [nextcloud/spreed#5352]
### Steps to reproduce
1. Klick on Talk
2. The App "opens"

### Expected behaviour
The app should show up.

### Actual behaviour
An empty, white screen appears - nothing else
The Nextcloud title bar including triangle next to the talk symbol is there, but nothing inside the window...

## Talk app

**Talk app version:** 9.0.9

**Custom Signaling server configured:** no

**Custom TURN server configured:** no

**Custom STUN server configured:** no

## Browser

**Microphone available:** yes

**Camera available:** yes

**Operating system:** Windows 10 Pro, German, Build 19042

**Browser name:** Firefox

**Browser version:** 86.0 (64-Bit)

### Browser log

I don't know where I can find these logs...

## Server configuration detail

**Operating system:** Linux 4.9.0-8-amd64 #1 SMP Debian 4.9.110-3+deb9u6 (2018-10-08) x86_64

**Webserver:** Apache (apache2handler)

**Database:** mysql 10.2.25

**PHP version:** 

7.4.15
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, session, standard, sodium, apache2handler, mysqlnd, PDO, xml, apcu, bcmath, bz2, calendar, ctype, curl, dba, dom, enchant, mbstring, FFI, fileinfo, ftp, gd, gettext, gmp, iconv, imagick, imap, intl, json, ldap, exif, mysqli, odbc, pdo_dblib, PDO_Firebird, pdo_mysql, PDO_ODBC, pdo_pgsql, pdo_sqlite, pgsql, apc, posix, pspell, readline, shmop, SimpleXML, snmp, soap, sockets, sqlite3, sysvmsg, sysvsem, sysvshm, tidy, tokenizer, xmlreader, xmlrpc, xmlwriter, xsl, zip, Phar, ionCube Loader, Zend OPcache

**Nextcloud version:** 19.0.9 - 19.0.9.1

**Updated from an older Nextcloud/ownCloud or fresh install:** Continuously updated

**Where did you install Nextcloud from:** unknown

<details><summary>Signing status</summary>

Integrity checker has been disabled. Integrity cannot be verified.
</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - activity: 2.12.1
 - admin_audit: 1.9.0
 - announcementcenter: 3.8.1
 - apporder: 0.12.0
 - audioplayer: 2.13.1
 - bookmarks: 3.4.9
 - bruteforcesettings: 2.0.1
 - calendar: 2.0.5
 - cloud_federation_api: 1.2.0
 - comments: 1.9.0
 - contacts: 3.4.3
 - contactsinteraction: 1.0.0
 - dav: 1.15.0
 - external: 3.6.0
 - extract: 1.3.1
 - federatedfilesharing: 1.9.0
 - federation: 1.9.0
 - files: 1.14.0
 - files_downloadactivity: 1.8.0
 - files_external: 1.10.0
 - files_pdfviewer: 1.8.0
 - files_rightclick: 0.16.0
 - files_sharing: 1.11.0
 - files_texteditor: 2.14.0
 - files_trashbin: 1.9.0
 - files_versions: 1.12.0
 - files_videoplayer: 1.8.0
 - flowupload: 1.1.2
 - gallery: 18.5.0
 - gpxpod: 4.2.2
 - group_everyone: 0.1.7
 - groupfolders: 7.1.1
 - impersonate: 1.6.1
 - issuetemplate: 0.7.0
 - logreader: 2.4.0
 - lookup_server_connector: 1.7.0
 - metadata: 0.13.0
 - nextcloud_announcements: 1.8.0
 - notifications: 2.7.0
 - oauth2: 1.7.0
 - previewgenerator: 2.3.0
 - privacy: 1.3.0
 - provisioning_api: 1.9.0
 - serverinfo: 1.9.0
 - settings: 1.1.0
 - sharebymail: 1.9.0
 - spreed: 9.0.9
 - systemtags: 1.9.0
 - tasks: 0.13.6
 - text: 3.0.1
 - theming: 1.10.0
 - twofactor_backupcodes: 1.8.0
 - updatenotification: 1.9.0
 - viewer: 1.3.0
 - workflowengine: 2.1.0
Disabled:
 - accessibility
 - encryption
 - firstrunwizard
 - mail
 - password_policy
 - photos
 - polls
 - recommendations
 - support
 - survey_client
 - user_ldap
 - user_usage_report

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "***REMOVED SENSITIVE VALUE***",
    ],
    "activity_expire_days": "120",
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "filelocking.debug": false,
    "filelocking.enabled": true,
    "filelocking.ttl": 600,
    "overwrite.cli.url": "***REMOVED SENSITIVE VALUE***",
    "version": "19.0.9.1",
    "installed": true,
    "dbtype": "mysql",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbtableprefix": "oc_",
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "default_language": "de",
    "defaultapp": "files",
    "integrity.check.disabled": true,
    "knowledgebaseenabled": true,
    "enable_avatars": true,
    "allow_user_to_change_display_name": true,
    "theme": "",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "loglevel": "2",
    "logdateformat": "Y-m-d H:i:s",
    "logtimezone": "Europe\/Berlin",
    "log_rotate_size": 20971520,
    "skeletondirectory": "\/home\/www\/nextcloud_data\/skeleton",
    "updatechecker": true,
    "trashbin_retention_obligation": "auto",
    "versions_retention_obligation": "disabled",
    "appstoreenabled": true,
    "appstore.experimental.enabled": true,
    "singleuser": false,
    "maintenance": false,
    "memcache.local": "\\OC\\Memcache\\APCu",
    "htaccess.RewriteBase": "\/",
    "mail_smtpmode": "smtp",
    "updater.release.channel": "stable",
    "mail_smtpsecure": "ssl",
    "mail_smtpauthtype": "LOGIN",
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpauth": 1,
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpport": "465",
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "mysql.utf8mb4": true,
    "log_type": "file",
    "app_install_overwrite": [
        "announcementcenter",
        "calendar",
        "files_downloadactivity",
        "apporder",
        "external",
        "impersonate"
    ],
    "jpeg_quality": "90",
    "share_folder": "\/",
    "simpleSignUpLink.shown": false,
    "data-fingerprint": ""***REMOVED SENSITIVE VALUE***"
}
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** 

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

## Client configuration

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.106 Safari/537.36

**Operating system:** Windows 10 Pro German, 64-Bit, Build 19042

## Logs

<details><summary>Web server error log</summary>

```
no error related to that problem
```
</details>

<details><summary>Nextcloud log</summary>

```
nothing related to talk at the log...
```
</details>

<details><summary>Browser log</summary>
14:20:30.198 Content Security Policy: Die Einstellungen der Seite haben das Laden einer Ressource auf inline blockiert ("script-src"). content-hooks-frames.js:63:42
14:20:30.852 JQMIGRATE: Migrate is installed, version 1.4.1 jquery-migrate.min.js:2:551
14:20:30.875 $ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. 2 globals.js:61:15
14:20:30.884 Handlebars is deprecated: please ship your own, this will be removed in Nextcloud 20 globals.js:61:15
14:20:30.894 $ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. 3 globals.js:61:15
14:20:30.896 Notifications permissions denied App.vue:296
14:20:30.938 OCA.Viewer initialized Viewer.js:41:10
14:20:30.943 No OCA.Files app found, viewer is now in standalone mode Viewer.vue:291
14:20:31.036 jQuery is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:61:15
14:20:31.039 $ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:61:15
14:20:31.419
load { target: HTMLDocument https://<SENSITIVE-DATA-REMOVED>/apps/spreed/, isTrusted: true, srcElement: HTMLDocument https://<SENSITIVE-DATA-REMOVED>/apps/spreed/, currentTarget: Window, eventPhase: 2, bubbles: false, cancelable: false, returnValue: true, defaultPrevented: false, composed: false, … }
content_script_bundle.js:95:594
14:20:31.128 Uncaught TypeError: navigator.userAgent.match(...) is null
    <anonymous> index-browser.js:7
    Webpack 13
index-browser.js:7:43
14:20:31.164 session heartbeat polling started session-heartbeat.js:97:9
14:20:31.168 $ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. 8 globals.js:61:15
14:20:31.592 $ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. 57 globals.js:61:15
</details>

**Comment by nickvergessen** [comment]
The browser log is accessible by pressing F12

**Comment by nickvergessen** [comment]
It sounds like you are iframing the website?

**Comment by Bodenseematze** [comment]
Huh?
Not that I know - if it is so, some app/tool does this?!

**Comment by nickvergessen** [comment]
> content-hooks-frames.js

this is not from Nextcloud and seems to be causing issues.

**Comment by nickvergessen** [comment]
One app/tool found by google is Zotero, are you using that?

**Comment by Bodenseematze** [comment]
> > content-hooks-frames.js
> 
> this is not from Nextcloud and seems to be causing issues.

I searched through all folders at my installation - that file was not found.
So I assume, it must coming from some Firefox browser plugin?

I tested the behaviour with Microsoft's Edge (the newer one which is based on Google's chrome engine) - with that Talk seems to work!

But I'v no idea, which plugin could cause such behaviour?!

> One app/tool found by google is Zotero, are you using that?
No, I never heard about that...
